index.html

```diff
@@ -90,6 +90,28 @@
 
 <div class="section section--latest">
   <div class="container">
+    <div class="partners">
+      <h2 class="partners__title">Partners</h2>
+      <div class="partners__body">
+        <div>
+          <a class="logo" href="https://datproject.org" target="_blank">
```

Comment:
In the future, I think we should only open links in a new tab if if opens video, audio, or the like.